Add Airbyte mcp to the ai developer zone (#695)
box/developer.box.com: content/pages/ai-dev-zone/index.md

        <strong style="background-color: #e8e8e8">Integration</strong>
    </Tile>
    <Tile type="box-brown" title="Airbyte"
      href="https://github.com/box-community/airbyte/blob/barduinor/source-box-devrel/docs/integrations/sources/box-data-extract.md">

>         Transform unstructured documents into structured, queryable data with  Airbyte source connector called “Box data extract”.

Reviewer: ```suggestion
        Transform unstructured documents into structured, queryable data by using the “Box data extract”, an Airbyte source connector.
```
